README.md

```diff
@@ -132,9 +132,9 @@ To set options and keep lint-staged extensible, advanced format can be used. Thi
 
 ## Filtering files
 
-It is possible to run linters for certain paths only by using [minimatch](https://github.com/isaacs/minimatch) patterns.
-The file patterns should be specified relative to the package root (where `lint-staged` is installed).
-The paths passed to the linters are absolute to avoid confusion in case they're executed with a different working directory.
+It is possible to run linters for certain paths only using glob patterns using [minimatch](https://github.com/isaacs/minimatch) to match those patterns. File patterns should be specified _relative to the `package.json` location_ (i.e. where `lint-staged` is installed).
```

Comment:
I think breaking up this sentence might be better.
> It is possible to run linters for certain paths only **using** glob _patterns_ **using** mini**match** to **match** those _patterns_.

How about this instead:
`It is possible to run linters for certain paths only by using glob patterns. [minimatch](https://github.com/isaacs/minimatch) is used to filter the staged files according to these patterns.`

Reply:
I like it!